python/lsst/cp/pipe/linearity.py

```diff
@@ -31,26 +31,37 @@
 from lsst.ip.isr import (Linearizer, IsrProvenance)
 
 from .utils import (funcPolynomial, irlsFit)
-
+from ._lookupStaticCalibration import lookupStaticCalibration
 
 __all__ = ["LinearitySolveTask", "LinearitySolveConfig", "MeasureLinearityTask"]
 
 
 class LinearitySolveConnections(pipeBase.PipelineTaskConnections,
-                                dimensions=("instrument", "detector")):
-    inputPtc = cT.Input(
-        name="inputPtc",
-        doc="Input PTC dataset.",
-        storageClass="StructuredDataDict",
-        dimensions=("instrument", "detector"),
-        multiple=False,
+                                dimensions=("instrument", "exposure", "detector")):
+    dummy = cT.Input(
+        name="raw",
+        doc="Dummy exposure.",
+        storageClass='Exposure',
+        dimensions=("instrument", "exposure", "detector"),
+        multiple=True,
+        deferLoad=True,
     )
-    camera = cT.Input(
+    camera = cT.PrerequisiteInput(
         name="camera",
         doc="Camera Geometry definition.",
         storageClass="Camera",
         dimensions=("instrument", ),
+        isCalibration=True,
+        lookupFunction=lookupStaticCalibration,
     )
+    inputPtc = cT.PrerequisiteInput(
```

Comment:
Before, this task wasn't using as input the output of the PTC?

Reply:
It was, but it wasn't a `PrerequisiteInput`.  I also rearranged the connections in the class to match how it was in `makeBrighterFatterKernel.py`.